index.bs

```diff
@@ -0,0 +1,139 @@
+<pre class='metadata'>
+Title: Audio Session
+Shortname: audio-session
+Level: None
+Status: w3c/ED
+Group: mediawg
+TR: https://www.w3.org/TR/audio-session/
+Repository: w3c/audio-session
+URL: https://w3c.github.io/audio-session/
+Editor: Youenn Fablet, Apple https://www.apple.com/, youenn@apple.com, w3cid 96458
+Editor: Alastor Wu, Mozilla https://www.mozilla.org, alwu@mozilla.com, w3cid 92198
+Abstract: This API defines an API surface for manipulating the bits on
+Markup Shorthands: css no, markdown yes
+</pre>
+<pre class=link-defaults>
+spec:webidl; type:dfn; text:resolve
+</pre>
+
+# Introduction # {#introduction}
+
+People consume a lot of media (audio/video) and the Web is one of the primary means of consuming this type of content.
+However, media on the web does not integrate well with the platform.
+The Audio Session API helps to close the gap with platforms that have audio session/audio focus such as Android and iOS.
+This API will help by improving the audio-mixing of websites with native apps, so they can play on top of each other, or play exclusively.
+
+Additionally, on some platforms the user agent will automatically manage the audio session for the site
```

Comment:
I wonder why the user agent can't define the default behaviors on all platforms, or you was specifically referring the part of interacting with the native app which requires the native platform API support? Otherwise, for handling the audio competing inside browser, they should/could be the same among all platforms.

Reply:
This sentence is about heuristics implemented by UA in case it runs on a platform that has an audio session like API. For instance, Safari is setting the session type to Ambient With WebAudio on iOS.
This is mostly targeted at the interaction with other native apps.
These heuristics are represented with `auto`.